Heroes Handbook, Main File.txt

```diff
@@ -1654,32 +1657,33 @@ Zheng, Yan, Wei Zhelan, Xuefeng, Li, Liuxian
 Keenwalker, Calmbrow, Mellowcoil, Ironshadow, Wildfur
 
 ### Pandaren Traits
-Pandaren has the following characteristics, harnessed through their immense training.
+Your pandaren character has certain traits deriving from their ancient heritage.
 
-***Ability Score Increase.*** Your Wisdom score increases by 2, and your Charisma score increases by 2.
+***Ability Score Increase.*** Your Wisdom score increases by 1, and your Charisma score increases by 2.
 
-***Age.*** Much like humans, pandaren reach adulthood in their late teens, and live approximately to become the same age as strong humans, most dying within a century.
+***Age.*** Pandaren age much like humans, reaching adult-hood in their late teens, and generally die before they reach their first century.
 
-***Alignment.*** Pandaren are a peaceful folk, and for the most part tend towards neutral or good, finding a pandaren that is of an evil alignment, is uncommon.
+***Alignment.*** Pandaren are a peaceful race, and for most parts tend to their own, making many pandarens neutral or good aligned. Finding an evil aligned pandaren is unusual.
 
-***Size.*** Pandaren are between 5 to 6 feet tall, and always on the chubby side, but never lets that hinder them or take offence from it. Your size is Medium.
+***Size.*** Most pandaren stands just short of 7 feet tall and average about 260 pounds, you will never find a pandaren who is 'slim'. Your size is medium.
 
 ***Speed.*** Your base walking speed is 30 feet.
 
-***Bouncy.*** You have resistance against bludgeoning damage taken from falling.
+***Gourmand.*** You gain proficiency with the artisan's tool (cook's utensils) and (brewer's supplies). 
+
+***Iron Palms.*** Your unarmed strikes deal 1d4 + Strength modifier bludgeoning damage on a hit.
 
-***Gourmand.*** You are proficient in the brewer's supplies and cook's utensils.
+***Pandaren Resilience.*** You have advantage on saving throws against poison, and you have resistance against poison damage.
 
-***Pandaren Stomach.*** You have advantage on saving throws against poison.
+***Pandaren Weapon Training.*** You have proficiency with the club, quarterstaff, and spear.
 
-***Quaking Palm.*** Upon hitting a target with an unarmed strike, you may transfer chi from your body to the targets, the target must succeed on a Constitution saving throw, or be stunned until the end of your next turn. Wisdom is your saving throw ability for this feature. You can't use this fea-ture again until you finish a short or long rest.
+***Quaking Palm.*** When you hit another creature with an unarmed strike, you may transfer chi from your body to the targets, the target must succeed on a Constitution saving throw, or be stunned until the end of your next turn. Wisdom is your saving throw ability for this feature. You can't use this feature again until you finish a long rest.
```

Comment:
```suggestion
***Quaking Palm.*** When you hit another creature with an unarmed strike, you may transfer your chi from your body to the target. The target must succeed on a Constitution saving throw, or be stunned until the end of your next turn. Wisdom is your saving throw ability for this feature. You can't use this feature again until you finish a long rest.
```

Proposing a clarification, so people don't think Quakin Palm can affect multiple creature in a single move. It's one unarmed attack against one creature, for which they can choose to active it, right?